cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -442,7 +456,7 @@ def filter_cases_related_to_dna_sample(
             ):
                 if not case.is_uploaded:
                     LOG.warning(
-                        f"Will not upload report to DNA case {case.internal_id} since its upload has not finshed."
+                        f"Will not upload report to DNA case {case.internal_id} since its upload has not finished."
```

Comment:
```suggestion
                        f"Cannot upload RNA report to DNA case {case.internal_id}. DNA case is not uploaded."
```

You don't know whether the process is running/on hold etc. The uploaded_at only says we cannot find an upload, maybe this won't let the user think something might be ongoing.